uiowa/uids
# Feature: menu level depth updates for the sidebar navigation

# How to test

- `menu-wrapper--horizontal` has been commented out until the container queries polyfill has been fixed the styles are now being set in `uiowa`. 
- Verify that `page-container` breakpoints still work, I changed `container` breakpoint to `page-container` to avoid namespace collision with container queries syntax. 
- Verify that `container-width` mixin works, I changed `container` mixin to `container-width` to avoid namespace collision with container queries syntax. 
- Verify that the sidebar menu adapts to 4 levels of depth.   If you adjust the levels it should look like this screenshot below:
- Review code changes
<img width="515" alt="Screen Shot 2022-02-01 at 2 47 14 PM" src="https://user-images.githubusercontent.com/1036433/152048770-36156e0b-5253-4ccd-a8b9-b2ca841903a7.png">
